modules/lambdas/main.tf

```diff
@@ -146,6 +147,55 @@ resource "aws_lambda_function" "extract_filepaths_for_granule" {
 }
 
 
+data "aws_iam_policy_document" "assume_lambda_role_extract" {
+  statement {
+    principals {
+      type = "AWS"
+      identifiers = ["*"]
+    }
+    actions = ["sts:AssumeRole"]
+
+  }
+}
+
+# Permissions needed to run the function successfully
+data "aws_iam_policy_document" "extract_filepaths_for_granule_policy_document" {
+  statement {
+    actions = [
+      "lambda:Invoke*",
+      "s3:PutObject*"
```

Comment:
are we adding anything to a bucket using that lambda? If not, I think this can be removed.

Reply:
The lambda does not work at all if "s3:PutObject" isn't defined, I tried using "s3:GetObject" but the function and step function immediately fail.

Reply:
okay then that makes sense to add it